README.md

```diff
@@ -1,190 +1,59 @@
-framework provides test executives with a single integration point for all
-OCPDiag compatible diagnostics. The goal is to eliminate the wrapping effort
-required in porting other diagnostics to different execution environments and
-result schemas.
+## What problems does it address?
 
-For diagnostics that run on multi-node systems, OCPDiag offers configuration
-options for targeting different endpoints and backends.
+* Acceleration/re-use of diagnostic development and integration efforts at all stages of the product life-cycle.
+* Diagnostic portability across multiple products, environments, and use-cases.
+* Reproduction of test and validation issues across multiple hardware and software partners.
+* Simple sharing of component vendor tests to accelerate RMA and root-cause analysis.
 
-## Overview
+## What does this project provide?
 
-OCPDiag diagnostics can run on a device under test (DUT) or off the DUT. After a
-OCPDiag diag is built and installed, a test executive or a human needs to start
-the test and collect its results.
+* Proven thoroughly documented data model for diagnostic output
+* API’s to easily produce that output.
+* Streaming results for long running tests
+* Simple but capable parameter management
+* An optional device communication library
+* An optional hardware interface layer
 
-![OCPDiag workflow](/ocpdiag/g3doc/ocpdiag_workflow.png)
+## What are the different repos making up this effort?
 
-OCPDiag focuses on three main topics: Parameter Model, Results Model, and
-Hardware Interface.
+This repo contains the diagnostic core libraries that are compatible with C++ and Python. These libraries provide the framework to reliably generated diagnostics that are compatible with clients designed to consume the OCP diagnostic output format.  It also contains the detailed specification covering the schema of the diagnostic output and guidance on how it should be leveraged.  This schema has been designed from the ground up to provide the necessary meta data and structure to be used effectively in heuristic and ML based systems for hardware troubleshooting and repair activities.  It has been proven to be a key enabler of diagnosability and repair at scale for large fleets of data center hardware when combined with downstream repair automation systems.
 
-This documentation provides schema specificaton, reference implementation, and
-example usage for each of these topics. Each set of the core libraries can be
-imported individually without affecting the usages of other subdomains.
+In this repo you will find:
 
-The following paragraphs use a [simple example](ocpdiag/core/examples/simple/)
-to introduce the build rules, the models, and hardware interface.
+* *apis* - These are implementations of the OCP diagnostic framework in c++ with python bindings, as well as a pure python implementation.
+* *ci_scripts* - repo automation for pre-submit testing for this repository.
```

Comment:
this is prob not worth mentioning here; it's an internal usage thing. maybe we should add a section/doc on "how to contribute" and add this there, since it's a dev aspect

Reply:
Removed.